src/errors.rs

```diff
@@ -85,6 +96,30 @@ impl Error {
             ErrorPos((*pos.filename).clone(), pos.line, pos.line_offset),
             message).into()
     }
+
+    pub fn custom_error<T: StdError + 'static>(err: T)
```

Comment:
Why `custom_error`. I think it's `custom` everywhere in the rust ecosystem?

Reply:
Because of implementing `custom` at `impl ::serde::de::Error for Error`:

```
error[E0277]: the trait bound `std::string::String: std::error::Error` is not satisfied                   
  --> src/de.rs:73:28                                                                                     
   |                                                 
73 |                 return Err(Error::custom(format!("bad boolean {:?}", self.0)));                      
   |                            ^^^^^^^^^^^^^ the trait `std::error::Error` is not implemented for `std::string::String`                                                                                            
   |                                                                                                      
   = note: required by `errors::Error::custom`
```

Reply:
You can write it as `de::Error::custom` instead. Presumably, this will never happen outside of the quire library itself (i.e. having `de::Error` in scope as well as `quire::Error`)